package.json

```diff
@@ -40,13 +41,14 @@
     "jest": "node ./packages/jest-cli/bin/jest.js",
     "jest-coverage": "yarn run jest -- --coverage",
     "lint": "eslint . --cache",
+    "lint-docs": "eslint --rule \"{'no-undef': 0, 'consistent-return': 0, 'no-unused-vars': 0}\" --no-ignore --ext md ./docs/*.md",
     "postinstall": "node ./scripts/postinstall.js && node ./scripts/build.js && (cd packages/eslint-plugin-jest && yarn link) && yarn link eslint-plugin-jest",
     "publish": "yarn run build-clean && yarn run build && lerna publish",
     "test-ci": "yarn run typecheck && yarn run lint && yarn run build && yarn run jest-coverage -- -i && yarn run test-examples && node scripts/mapCoverage.js && codecov",
     "test-ci-partial": "yarn run build && yarn run jest -- -i && yarn run test-examples",
     "test-examples": "node scripts/test_examples.js",
     "test-pretty-format-perf": "node packages/pretty-format/perf/test.js",
-    "test": "yarn run typecheck && yarn run lint && yarn run build && yarn run jest && yarn run test-examples",
+    "test": "yarn run typecheck && yarn run lint && yarn run build && yarn run jest && yarn run test-examples && yarn run lint-docs",
```

Comment:
we should probably put it after `yarn run lint`. You don't want to wait for the entire test suite to run to find out that you have 1 lint error in the docs :)